src/modules/auth/auth.service.ts

```diff
@@ -154,11 +141,17 @@ export default class AuthService {
 
     const accessToken = this.jwtService.sign(
       { ...doctor },
-      { expiresIn: '8h' },
+      { expiresIn: '24h' },
     );
 
-    res.cookie('accessToken', accessToken, this.accessTokenCookieOptions);
-    res.redirect(this.configService.get('CLIENT_URL'));
+    res.clearCookie('accessToken');
+    res.cookie('accessToken', accessToken);
+
+    res.redirect(this.configService.get('SECOND_FORM_URL'));
+
+    // res.redirect(this.configService.get('CLIENT_API'));
```

Comment:
why we need cookie? I dont like this solution as server and frontend will be on different servers, its mean that cookie will not work